checker/jtreg/nullness/defaultsPersist25/ReferenceInfoUtil.java

```diff
@@ -0,0 +1,100 @@
+import java.lang.classfile.*;
+import java.lang.classfile.attribute.*;
+import java.util.*;
+
+public class ReferenceInfoUtil {
```

Comment:
Can you remind me why we have Driver and ReferenceInfoUtil in each folder? Can you document the reason in the Javadocs?